.github/workflows/cron-issue.yaml

```diff
@@ -4,28 +4,22 @@ on:
   push:
     branches: [ main ]
   schedule:
-    - cron:  '0 4,16 * * *'
+    # Create an issue every day on 00:01 CST
+    - cron:  '1 16 * * *'
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [ '3.10' ]
-    name: Cron Create Issue
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup python
-        uses: actions/setup-python@v2
+      - name: Get today's date
+        id: date
+        run: |
+          echo "::set-output name=today::$(date "+%Y-%m-%d")"
```

Comment:
The local time is based on CST/UTC+0 time zone.

So when the cronjob run `2021-11-02 00:01:00 UTC+8`, the output day will be `2021-11-01`

I have revert the PR first